ovos_utils/enclosure/__init__.py

```diff
@@ -28,6 +48,11 @@ def enclosure2rootdir(enclosure=None):
         return MycroftRootLocations.OVOS
     elif enclosure == MycroftEnclosures.BIGSCREEN:
         return MycroftRootLocations.BIGSCREEN
+    elif find_root_from_sys_path():
+        return find_root_from_sys_path()
+    elif find_root_from_sitepackages():
+        return find_root_from_sitepackages()
```

Comment:
this should not return None, i'm unsure on how to best handle missing detections,

returning None probably makes sense, in which case since you are touching it maybe the default return value of MycroftRootLocations.HOME should also be changed to None


the only internal usage is in configuration.py

```python
class MycroftDefaultConfig(ReadOnlyConfig):
    def __init__(self):
        path = join(enclosure2rootdir(), "mycroft",
                    "configuration", "mycroft.conf")
        super().__init__(path)
        if not self.path or not isfile(self.path):
            LOG.error("mycroft root path not found, could not load default "
                      ".conf")
```

Reply:
looking back this seems like a brain fart of mine, search_mycroft_core_location should be used instead of enclosure2rootdir in the config